onnxruntime/core/providers/cpu/controlflow/event_pool.h

```diff
@@ -21,7 +21,9 @@ class OrtEventPool final {
   }
   void SignalEvent(int64_t id);
   bool QueryEvent(int64_t id) const;
+  void WaitAndResetEvent(int64_t id);
```

Comment:
Changes in this file can be reverted. 
and this can be moved back to orttraining location.